fastlane/Fastfile

```diff
@@ -17,4 +17,69 @@ platform :ios do
       test | xcpretty
       ")
   end
+
+  desc "Runs the test suite and calculates code coverage percentages for each
+  module within the SDK"
+  lane :coverage do
+    MINIMUM_COVERAGE = 100.0
```

Comment:
This was an assumption on my part

Reply:
I like the 80% per DoD, curious to hear other thoughts :smile:

Reply:
If we wanted to try 100%, now is the time, then we can re-evaluate and go down a bit if we feel like 100% isn't necessary? 🤷‍♂️

Reply:
80% works for me!